### PR TITLE
Danbooru plg

### PR DESCRIPTION
**Pull request by AckiyBolt**
Gets a random fresh or popular image from Danbooru
Because there are mostly hentai content - 18+

### CODE REVIEW

File: plugins/dan.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+local scale_day = "?scale=day"
+local scale_week = "?scale=week"
+local scale_month = "?scale=month"
+
+function get_post(url)
+  local b, c, h = http.request(url)
+  local posts = json:decode(b)
+
+  -- bad random - bad magic...
+  math.randomseed( os.time() )
```
Comment (yagop) [inline plugins/dan.lua]: The `randomseed` is started on https://github.com/yagop/telegram-bot/blob/master/bot/bot.lua#L235
I can merge and remove yours.

Comment (AckiyBolt) [inline plugins/dan.lua]: oh. ok
i miss that line

**Comment by Akamaru** [comment]
Hi
is it possible to add a search for tags?

**Comment by AckiyBolt** [comment]
Hi
If u talk about search for random image by one or few tags - i think it's possible. But tags should be exactly are they are

**Comment by Akamaru** [comment]
Yes that's what I mean. That would be nice

**Pull request merged**